state/state.go

```diff
@@ -926,7 +927,33 @@ func (s *State) LoadDBState(dbheight uint32) (interfaces.IMsg, error) {
 		panic(fmt.Sprintf("The configured network ID (%x) differs from the one in the local database (%x) at height %d", configuredID, dbaseID, dbheight))
 	}
 
-	msg := messages.NewDBStateMsg(s.GetTimestamp(), dblk, ablk, fblk, ecblk, eBlocks, entries)
+	blockSig := new(primitives.Signature)
+	var allSigs []interfaces.IFullSignature
+
+	nextABlock, err := s.DB.FetchABlockByHeight(dbheight + 1)
+	if err != nil || nextABlock == nil {
+		return nil, err
```

Comment:
This is where you can look in the process list to see if you can get the signatures there.